lib/refinement/cocoapods_post_install_writer.rb

```diff
@@ -3,16 +3,17 @@ module Refinement
   # takes into account changes to Pod configuration,
   # as well as the globs used by podspecs to search for files
   class CocoaPodsPostInstallWriter
-    attr_reader :aggregate_targets, :config, :repo, :options
-    private :aggregate_targets, :config, :repo, :options
+    attr_reader :aggregate_targets, :pod_targets, :config, :repo, :options
+    private :aggregate_targets, :pod_targets, :config, :repo, :options
 
     # Initializes a post-install writer with CocoaPods target objects.
     # @return [CocoaPodsPostInstallWriter] a new instance of CocoaPodsPostInstallWriter
     # @param aggregate_targets [Array<Pod::AggregateTarget>]
     # @param config [Pod::Config]
     # @param options [Hash]
-    def initialize(aggregate_targets, config, options)
+    def initialize(aggregate_targets, pod_targets, config, options)
```

Comment:
argh forgot to comment, missed parameter in the method doc.

Reply:
https://github.com/square/refinement/pull/10